packages/app/src/app/overmind/createConnect.ts

```diff
@@ -0,0 +1,248 @@
+import { context } from './Provider';
+import { IConfiguration, Overmind, EventType } from 'overmind';
+import {
+  StatelessComponent,
+  ComponentClass,
+  ClassicComponentClass,
+  Component,
+  createElement,
+} from 'react';
+
+export type IReactComponent<P = any> =
+  | StatelessComponent<P>
+  | ComponentClass<P>
+  | ClassicComponentClass<P>;
+
+// Diff / Omit taken from https://github.com/Microsoft/TypeScript/issues/12215#issuecomment-311923766
+type Omit<T, K extends keyof T> = Pick<
```

Comment:
@christianalfoni `Omit` is included in the core of TypeScript since 3.5
https://devblogs.microsoft.com/typescript/announcing-typescript-3-5